modules/docs/arrow-docs/docs/docs/arrow-fx/queue/README.md

```diff
@@ -0,0 +1,213 @@
+---
+layout: docs
+title: Queue
+permalink: /docs/effects/queue/
+---
+
+## Queue
+
+{:.intermediate}
+intermediate
+
+A `Queue` is a lightweight, asynchronous first-in-first-out queue for holding arbitrary values within a `Concurrent`
+context. There are two primary means of working with an instance of a `Queue`: placing items via `offer` and removing
+items via `take`.
+
+```kotlin:ank:playground
+import arrow.fx.*
+import arrow.fx.extensions.fx
+import arrow.fx.extensions.io.concurrent.concurrent
+
+fun main(args: Array<String>) {
+  val result =
+  //sampleStart
+    IO.fx {
+      val q = !Queue.bounded<ForIO, Int>(10, IO.concurrent()).fix()
```

Comment:
Shouldn't be necessary to fix here since `!` is defined for `Kind<ForIO, A>`.

```suggestion
      val q = !Queue.bounded<ForIO, Int>(10, IO.concurrent())
```